helpers/mathjax.js

```diff
@@ -129,12 +113,18 @@ export function loadMathJax(mathJaxConfig) {
 				const AbstractHandler = window.MathJax._.core.Handler.AbstractHandler.prototype;
 				const startup = window.MathJax.startup;
 
+				const getFirstChild = doc => {
+					const child = doc.firstChild;
+					if (!child || child.nodeType === Node.ELEMENT_NODE) return child;
+					else return child.nextElementSibling;
+				};
```

Comment:
This handles the Lit comment problem. The reason that exploded in our custom adapter here is because the adaptor is expecting a structure that looks exactly like this:
```
<div class="d2l-html-block-rendered">
   <mjx-doc>
      <mjx-head>...</mjx-head>
      <mjx-body>...</mjx-body>
   </mjx-doc>
</div>
```
Effectively, we use these `mjx-{structure}` nodes to simulate an HTML document for MathJax to process. Lit messes things up here because it produces a structure like this (note the comment):
```
<div class="d2l-html-block-rendered">
   <!--lit$1425314-->
   <mjx-doc>...</mjx-doc>
</div>
```
So the solution implemented here is to just deliberately ignore non-element nodes when our adaptor parses the document structure.

Reply:
Actually, this isn't true now that I'm wiping out the container's inner HTML before stamping. Let me see if I can remove this.

Reply:
I think the comment node is what was blowing up when I first tried to use Lit rendering with MathJax in the very beginning.  Glad we found a solution.  :)

Reply:
It feels a bit fragile, in the sense that if Lit ever decides to put comments elsewhere this might explode... But one of the things that appealed to me about MathJax 3.2 is that they added proper handling for comments _within_ the document body they're parsing. So I think this special handling for direct children of the shadowRoot is all we need.